### ISSUE TITLE
Server Crash - NPE: Ticking GridNode - Import bus on Interface

### ISSUE THREAD

**Issue by CarsonF** [ExtraCells/ExtraCells2#411]
Server crashed on placing the import bus. Interface has not been configured to export any fluid.

```
java.lang.NullPointerException: Ticking GridNode
    at extracells.part.PartFluidInterface.tickingRequest(PartFluidInterface.java:983)
    at appeng.me.cache.TickManagerCache.onUpdateTick(TickManagerCache.java:92)
```

[Full crash report](http://pastebin.com/rANYaYPJ)

![screen shot 2016-07-06 at 5 42 16 pm](https://cloud.githubusercontent.com/assets/932566/16637009/72822014-43a1-11e6-99b9-396365bc4e22.png)

**Comment by v1993** [comment]
I've got same bug.

**Comment by Nedelosk** [comment]
Fixed in 1.10.
